back/routes/adventure.js

```diff
@@ -34,19 +34,18 @@ router.post('/preferences/active', async(req, res) => {
 })
 
 // ----- Get all existing Preferences -----
-router.get('/preferences/all', async(req, res) => {
+router.post('/preferences/all', async(req, res) => {
```

Comment:
This should remain as a `GET` endpoint, right?

Reply:
I was actually having questions about get, because I could not submit a req.body as usual. I will definitely change that later, I just need time to find the right documentation for that